scripts/generate_llms_standard.py

```diff
@@ -0,0 +1,333 @@
+# Project-specific input
+docs_repo = 'moonbeam-docs'
+docs_url = 'https://docs.moonbeam.network/'
+docs_org = 'moonbeam-foundation'
```

Comment:
we should use the config file to define these, this will be necessary when we have a single repo for these. doesn't need to happen now, but so it's on your radar (although it probably already is)